core/server/data/migrations/versions/4.38/2022-03-01-15-42-update-products-visible-column.js

```diff
@@ -0,0 +1,36 @@
+const logging = require('@tryghost/logging');
+const {createTransactionalMigration} = require('../../utils');
+
+module.exports = createTransactionalMigration(
+    async function up(knex) {
+        const portalProductSetting = await knex('settings').select('value').where('key', 'portal_products').first();
+
+        if (!portalProductSetting) {
+            logging.warn('Could not find portal_products setting - skipping migration');
+            return;
+        }
+
+        try {
+            const settingData = JSON.parse(portalProductSetting.value);
+
+            if (settingData.length === 0) {
+                logging.warn(`portal_product is empty, skipping migrations`);
+                return;
+            }
+
+            logging.info(`Updating ${settingData.length} products to visible, ${settingData}`);
+            await knex('products').update('visible', true).whereIn('id', settingData);
+        } catch (err) {
```

Comment:
Can we log the `err` out here? If we don't, we won't know if it failed because the JSON.parse failed, or the DB update failed etc